addOns/help/src/main/javahelp/contents/releases/2.12.0.html

```diff
@@ -73,6 +149,7 @@ <H2>Add-Ons</H2>
 <H3>New Add-Ons</H3>
 The following add-ons are included by default in this release for the first time:
 <ul>
+  <li><a href="https://www.zaproxy.org/docs/desktop/addons/database/">Database</a> - provides database engines and related infrastructure..</li>
```

Comment:
Sorry to be a pain....drop a period? 😀

Reply:
Ha you saw it! 🥇

Reply:
I did, right after I pushed the prev changes 😆